driver-portal/src/ClientApp/src/app/submission-requirements/submission-requirements.component.html

```diff
@@ -45,128 +45,139 @@
         File button Certain documents cannot be uploaded (e.g., example here).
         Learn how to submit additional documentation.
         <form>
-        <div style="padding-top: 24px">
-          <mat-form-field appearance="outline" style="width: 350px">
-            <mat-label>Select Document Type</mat-label>
-            <mat-select name="documentType" [formControl]="uploadForm.controls.documentSubType">
-              @for (documentType of documentSubTypes; track documentType) {
-              <mat-option [value]="documentType.id">{{
-                documentType.name
-              }}</mat-option>
+          <div style="padding-top: 24px">
+            <mat-form-field appearance="outline" style="width: 350px">
+              <mat-label>Select Document Type</mat-label>
+              <mat-select
+                name="documentType"
+                [formControl]="uploadForm.controls.documentSubType"
+              >
+                @for (documentType of documentSubTypes; track documentType) {
+                <mat-option [value]="documentType.id">{{
+                  documentType.name
+                }}</mat-option>
+                }
+              </mat-select>
+              @if (uploadForm.controls.documentSubType.getError('required')) {
```

Comment:
same here, could use ngIf

Reply:
I have been using the new control flows in angular where Ngif, Ngfor are replaced with @if, @for  conditional expression from angular 16 https://angular.io/guide/control_flow